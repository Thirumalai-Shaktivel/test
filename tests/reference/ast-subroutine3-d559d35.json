{
    "basename": "ast-subroutine3-d559d35",
    "cmd": "lfortran --show-ast {infile} -o {outfile}",
    "infile": "tests/subroutine3.f90",
    "infile_hash": "a524604ef3f17ed8a7b81b5ac3b03932cbc3ec6f308982532b4d9900",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-subroutine3-d559d35.stdout",
    "stdout_hash": "7d28bf5f8b09b159f675afdf6d7cc763bc29e1fbaa7a9808d74e4b64",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}