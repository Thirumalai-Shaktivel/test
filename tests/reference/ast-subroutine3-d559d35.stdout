(Function f [] () () () [] [] [(= f 42)] [])
